src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java

```diff
@@ -19062,6 +19062,116 @@ public void run3() {
+                    200,
+                    480);
+            GT_Values.RA.addAssemblerRecipe(
+                    new ItemStack[] {
+                        GT_OreDictUnificator.get(OrePrefixes.bolt, Materials.EnderEye, 4),
+                        GT_OreDictUnificator.get(OrePrefixes.foil, Materials.Manganese, 4),
+                        GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Glowstone, 4),
+                        GT_Utility.getIntegratedCircuit(1)
+                    },
+                    Materials.EnergeticSilver.getMolten(576),
+                    GT_ModHandler.getModItem("EnderIO", "itemBasicCapacitor", 1L, 9),
+                    200,
+                    480);
+            GT_Values.RA.addAssemblerRecipe(
+                    new ItemStack[] {
+                        GT_OreDictUnificator.get(OrePrefixes.bolt, Materials.Platinum, 4),
+                        GT_OreDictUnificator.get(OrePrefixes.foil, Materials.Manganese, 4),
+                        ItemList.QuantumEye.get(1L),
+                        GT_Utility.getIntegratedCircuit(1)
+                    },
+                    Materials.CrystallineAlloy.getMolten(864),
+                    GT_ModHandler.getModItem("EnderIO", "itemBasicCapacitor", 1L, 3),
+                    200,
+                    1920);
+            GT_Values.RA.addAssemblerRecipe(
+                    new ItemStack[] {
+                        GT_OreDictUnificator.get(OrePrefixes.bolt, Materials.Titanium, 4),
+                        GT_OreDictUnificator.get(OrePrefixes.foil, Materials.Manganese, 4),
+                        ItemList.QuantumEye.get(1L),
+                        GT_Utility.getIntegratedCircuit(1)
+                    },
+                    Materials.CrystallineAlloy.getMolten(864),
+                    GT_ModHandler.getModItem("EnderIO", "itemBasicCapacitor", 1L, 3),
+                    200,
+                    1920);
+            GT_Values.RA.addAssemblerRecipe(
+                    new ItemStack[] {
+                        GT_OreDictUnificator.get(OrePrefixes.bolt, Materials.TungstenSteel, 4),
+                        GT_OreDictUnificator.get(OrePrefixes.foil, Materials.Manganese, 4),
+                        new ItemStack(Items.nether_star, 1, 1),
+                        GT_OreDictUnificator.get(OrePrefixes.plate, Materials.EndSteel, 1),
+                        GT_Utility.getIntegratedCircuit(1)
+                    },
+                    Materials.MelodicAlloy.getMolten(1152),
+                    GT_ModHandler.getModItem("EnderIO", "itemBasicCapacitor", 1L, 4),
+                    200,
+                    7680);
+            GT_Values.RA.addAssemblerRecipe(
+                    new ItemStack[] {
+                        GT_OreDictUnificator.get(OrePrefixes.bolt, Materials.VanadiumGallium, 4),
+                        GT_OreDictUnificator.get(OrePrefixes.foil, Materials.Manganese, 4),
+                        ItemList.Gravistar.get(1L),
```

Comment:
Neutronium is too late, it needs a uv ebf.

Reply:
> > Hm, aren't those foil mv ? why neutronium?....
> 
> that's stellar capactior's recipe

well so luv or zpm iirc?

Reply:
where do you see Neutronium ?

Reply:
The gravistar was there already if you mean this. Suggest another item

Reply:
gravi star need Molten Neutronium, 288L

Reply:
Quantum Star is enough, you don't seem to be using it.

Reply:
changed